src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -5,6 +5,33 @@ import {
 } from '@/interfaces'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
 
+/** Amalgamation (regular) resource interface. */
+export interface AmalgamationRegResourceIF {
+  entityType: CorpTypeCd
+  displayName: string
+  steps: Array<StepIF>
+  filingData: Array<FilingDataIF>
+  peopleAndRoles: PeopleAndRolesResourceIF
+  reviewAndConfirm: {
+    completingPartyStatement: CompletingPartyStatementIF
+  }
+
+  shareClasses?: {
+    countMinimum: number
+  }
+  incorporationArticles?: {
+    articles: string,
+    articlesTooltip: string,
+    provisions?: string,
+    provisionTooltip?: string
+  }
+  incorporationAgreement?: {
+    helpSection: Array<HelpSectionIF>
+    article: string
+    documents: Array<IncorporationAgreementTypeIF>
+  }
```

Comment:
I'm waiting to hear back on whether we need incorporationArticles or incorporationAgreement. I will look into this later.